client/src/shared/components/InputField/index.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import React from 'react';
+
+interface InputFieldProps {
+  label?: string;
+  placeholder?: string;
+  type?: string;
+  id?: string;
+  widthProp?: string;
+  heightProp?: string;
+}
+
+const InputField: React.FC<InputFieldProps> = ({
+  label,
+  placeholder,
+  type,
+  id,
+  widthProp,
+  heightProp
```

Comment:
- no need to add postfix `prop`.
```suggestion
  width,
  height
```